Source/Orts.Simulation/Common/Scripting/TrainControlSystem.cs

```diff
@@ -123,22 +136,26 @@ public abstract class TrainControlSystem : AbstractScriptClass
         /// </summary>
         public Func<float> SpeedMpS;
         /// <summary>
-        /// Train's direction.
+        /// Locomotive direction.
         /// </summary>
         public Func<Direction> CurrentDirection;
         /// <summary>
-        /// True if train direction is forward.
+        /// True if locomotive direction is forward.
         /// </summary>
         public Func<bool> IsDirectionForward;
         /// <summary>
-        /// True if train direction is neutral.
+        /// True if locomotive direction is neutral.
         /// </summary>
         public Func<bool> IsDirectionNeutral;
         /// <summary>
-        /// True if train direction is reverse.
+        /// True if locomotive direction is reverse.
         /// </summary>
         public Func<bool> IsDirectionReverse;
         /// <summary>
+        /// Train direction.
+        /// </summary>
+        public Func<Direction> CurrentTrainMUDirection;
```

Comment:
How does "MU Direction" differ from "locomotive direction"? We should include a blurb about this.

Reply:
That's a good question. For sure they are two different variables :), Looking at the code I expect that they couldn't coincide for flipped locos or for locos run from rear cab (maybe backwards, or in explore mode, or during shunting). So I think it's good to have that info.

Reply:
Right, but then the call is named `TrainMUDirection`, which at first glance might apply to both the train and the locomotive. :-)

Also consider adding a call for the locomotive's direction, but returning a `Direction` enum instead of a boolean. That would make it even more clear that they are two separate pieces of information.

Reply:
There's already a CurrentDirection() call available returning an enum. IsDirectionReverse() is a legacy call which was created a few years ago.

Reply:
Totally missed that, thanks for pointing it out. I'll be satisfied if we add documentation clarifying the definitions of locomotive direction, train direction, and MU direction.

Reply:
I will add that to the script .cs file.